swift/test/Ice/location/AllTests.swift

```diff
@@ -13,14 +13,14 @@ func allTests(_ helper: TestHelper) throws {
 
     let communicator = helper.communicator()
 
-    let manager = try checkedCast(
-        prx: communicator.stringToProxy("ServerManager:\(helper.getTestEndpoint(num: 0))")!,
+    let manager = try makeProxy(
+        communicator: communicator,
+        proxyString: "ServerManager:\(helper.getTestEndpoint(num: 0))",
         type: ServerManagerPrx.self
-    )!
+    )
 
     let locator = uncheckedCast(prx: communicator.getDefaultLocator()!, type: TestLocatorPrx.self)
-
-    let registry = try checkedCast(prx: locator.getRegistry()!, type: TestLocatorRegistryPrx.self)!
+    let registry = try uncheckedCast(prx: locator.getRegistry()!, type: TestLocatorRegistryPrx.self)
```

Comment:
In C++ we got rid of `uncheckedCast` usage with the constructor, will we do the same with makeProxy?